doc/source/changes/version_0_27.rst.inc

```diff
@@ -3,6 +3,9 @@
 
 * renamed `Axis.translate` to `Axis.index` (closes :issue:`479`).
 
+* deprecated `reverse` argument of `sort_values`, `sort_axes`, `labelsofsorted` and `indicesofsorted` methods
```

Comment:
remove mention of `indicesofsorted` and `labelsofsorted` here (they did not support reverse in 0.26.1). Also need to update the changelog for those which still says implemented "reverse"